README.md

```diff
@@ -320,11 +326,11 @@ Now add the following content to your newly created `blog_page.html` file:
 Note the use of Wagtail's built-in `get_parent()` method to obtain the
 URL of the blog this post is a part of.
 
-From <http://127.0.0.1:8000/blog/>, click "Add a child page" again, and this time create a new "Blog Page".
+From <http://127.0.0.1:8000/blog/>, click "Add a child page" again, and this will automatically create a new blog page for you.
 
 Wagtail gives you full control over what kinds of content can be created under
 various parent content types. By default, any page type can be a child of any
-other page type.
+other page type. We added some code to automatically create a child page that matches the parent page because it makes the user experience a little cleaner and keeps editors from selecting the wrong page type.
```

Comment:
The phrase "a child page that matches the parent page" might be a bit ambiguous as it could mean that they're both of the same type.

```suggestion
other page type. We updated the models so that `BlogPage` is automatically selected when creating a page under a `BlogIndexPage`, and we disallowed the creation of a `BlogPage` under other page types. This makes the user experience more seamless and keeps editors from selecting the wrong page type.
```